### DISCUSSION TITLE
API for editing display name?

### DISCUSSION THREAD

**Discussion by pacohope** [mastodon/mastodon#22012] [Q&A] [answered]
Is there an API for editing display name?

**Comment by trwnh** [comment, marked as answer]
EDIT: https://docs.joinmastodon.org/methods/accounts/#update_credentials

https://docs.joinmastodon.org/client/intro/

**Reply by pacohope** [reply]
That's not it.

**Reply by trwnh** [reply]
https://docs.joinmastodon.org/methods/accounts/#update_credentials

**Reply by pacohope** [reply]
Thanks.

**Reply by JacksonChen666** [reply]
@pacohope You should choose this as the correct answer so that it is marked as answered.
